src/current/cockroachcloud/alerts-page.md

```diff
@@ -3,45 +3,88 @@ title: Alerts Page
 summary: The Alerts page allows you to toggle CockroachDB Cloud alerts and view alert history.
 toc: true
 docs_area: manage
-page_version: v23.1
 ---
 
-The **Alerts** page allows you to enable email alerts, send test alerts, and view the email recipients and alert history for your CockroachDB {{ site.data.products.cloud }} organization. To view the Alerts page, [log in](https://cockroachlabs.cloud/) and click **Alerts**.
+The **Alerts** page allows you to enable email alerts, send test alerts, and view the email recipients and alert history for your CockroachDB {{ site.data.products.cloud }} organization. To view the Alerts page, [log in](https://cockroachlabs.cloud/) and click **Alerts**. 
 
 {{site.data.alerts.callout_info}}
 The **Alerts** page is applicable for CockroachDB {{ site.data.products.dedicated }} clusters in your CockroachDB {{ site.data.products.cloud }} organization. For CockroachDB {{ site.data.products.serverless }} clusters in your organization, all [Org Administrators]({% link cockroachcloud/authorization.md %}#org-administrator-legacy) automatically receive email alerts when your cluster reaches 50%, 75%, and 100% of your [resource limits](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/architecture/glossary#resource-limits).
 {{site.data.alerts.end}}
 
-If alerts are enabled, CockroachDB {{ site.data.products.cloud }} sends alerts to [specified email recipients](#configure-alerts) when the following usage metrics are detected:
+## Automatic alerts
+
+### Cluster Maintenance
 
-**Cluster Maintenance:**
+[Org Administrators]({% link cockroachcloud/authorization.md %}#org-administrator-legacy) receive email alerts when: 
 
-- When a cluster is scheduled for [maintenance or a patch upgrade]({% link cockroachcloud/cluster-management.md %}#set-a-maintenance-window) that could temporarily impact the cluster's performance.
+- A cluster is scheduled for an automatic [patch version upgrade]({% link cockroachcloud/upgrade-policy.md %}#patch-version-upgrades) and again after the upgrade is complete. 
+- When a cluster is scheduled for [maintenance]({% link cockroachcloud/cluster-management.md %}#set-a-maintenance-window) that could temporarily impact the cluster's performance.
```

Comment:
Can you please specify - Alerts on maintenance windows requires explicit email recipient. If the user wants to be updated about maintenance windows, they will have to set an alert with the appropriate email for notification.

Reply:
Added a sub-section in the opt-in alerts section